src/MigrationLog/MigrationLogServiceProvider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Give\MigrationLog;
+
+use Give\ServiceProviders\ServiceProvider;
+use Give\Framework\Migrations\MigrationsRegister;
+use Give\MigrationLog\Migrations\CreateMigrationsTable;
+
+/**
+ * Class MigrationLogServiceProvider
+ * @package Give\MigrationLog
+ *
+ * @since 2.10.0
+ */
+class MigrationLogServiceProvider implements ServiceProvider {
+	/**
+	 * @inheritdoc
+	 */
+	public function register() {
+		global $wpdb;
+
+		$wpdb->give_log = "{$wpdb->prefix}give_migration";
```

Comment:
@alaca Does `s` missing from table name?

Reply:
@ravinderk `give_logs` already exists, so I cant create a table with the same name. Also, updating the table structure won't work because of the existing logs which should be migrated to the new table

Reply:
@alaca Got it 👍